sdk/eventhub/event-hubs/src/eventProcessor.ts

```diff
@@ -89,11 +103,46 @@ export class EventProcessor {
    *
    * @return {Promise<void>}
    */
-  async start(): Promise<void> {}
+  async start(): Promise<void> {
+    const partitionIds = await this._eventHubClient.getPartitionIds();
+    const partitionContext: PartitionContext = {
+      partitionId: partitionIds[0],
+      consumerGroupName: this._consumerGroupName,
+      eventHubName: this._eventHubClient.eventHubName
+    };
+    const partitionProcessor = this._partitionProcessorFactory(
+      partitionContext,
+      new CheckpointManager()
```

Comment:
Shouldnt checkpoint manager take partition context and partition manager in its constructor?

Reply:
We didn't implemented `CheckpointManager` yet. After the implementation I'll update the sample.